src/UserSettingsPage.h

```diff
@@ -166,6 +167,7 @@ private slots:
 private:
         // Layouts
         QVBoxLayout *topLayout_;
+        QFormLayout *formLayout_;
```

Comment:
Do we really need to formLayout and the mainLayout? That sounds a bit redundant. I'm also getting the following warning, which may or may not be related:

```
[2020-02-18 19:35:31.947] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.947] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.947] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.947] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.948] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.948] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.948] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.948] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.948] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.948] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.948] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.948] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.948] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.949] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.949] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
[2020-02-18 19:35:31.949] [qml] [warning] QLayout: Attempting to add QLayout "" to UserSettingsPage "", which already has a layout (:0, )
```

Reply:
I kept the mainlayout because it seems to have the setContentsMargins and margin math in the window resize. (whereas IIRC the QFormLayout didn't have that method available)

Reply:
It actually should have those methods available, see here: https://doc.qt.io/qt-5/qformlayout-members.html

Reply:
also, judging from the number of messages, I'm guessing this is coming from boxWrap somewhere. I'll check on it.

Reply:
The parent of the layouts isn't the userSettingsPage though, but the layout immediately above them, isn't it?